WindowsServerDocs/administration/windows-commands/gpt.md

```diff
@@ -22,22 +22,28 @@ On basic GUID partition table (gpt) disks, assigns the gpt attribute(s) to the p
+| attributes=<n> | Specifies the value for the attribute that you want to apply to the partition with focus. The gpt attribute field is a 64-bit field that contains two subfields. The higher field is interpreted only in the context of the partition ID, while the lower field is common to all partition IDs. Accepted values include:<br /><br />-   **0x0000000000000001**. Specifies that the partition is required by the computer to function properly.<br />-   **0x8000000000000000**. Specifies that the partition will not receive a drive letter by default when the disk is moved to another computer or when the disk is seen for the first time by a computer.<br />-   **0x4000000000000000**. Hides a partition's volume. That is, the partition will not be detected by the mount Manager.<br />-   **0x2000000000000000**. Specifies that the partition is a shadow copy of another partition.<br />-   **0x1000000000000000**. Specifies that the partition is read-only. This attribute prevents the volume from being written to.<br /><br />For more information about these attributes, see the attributes section at [create_PARTITION_PARAMETERS Structure](https://go.microsoft.com/fwlink/?LinkId=203812) (<https://go.microsoft.com/fwlink/?LinkId=203812>). |
+
+## Remarks
+
+- The EFI System partition contains only those binaries necessary to start the operating system. This makes it easy for OEM binaries or binaries specific to an operating system to be placed in other partitions.
+- A basic gpt partition must be selected for this operation to succeed. Use the **select partition** command to select a basic gpt partition and shift the focus to it.
+
+## <a name="BKMK_examples"></a>Examples
+
+  if you are moving a gpt disk to a new computer and would like to prevent that computer from automatically assigning a drive letter to the partition with focus, type:
```

Comment:
Cosmetic : start the sentence with uppercase
```suggestion
  If you are moving a gpt disk to a new computer and would like to prevent that computer from automatically assigning a drive letter to the partition with focus, type:
```